agora.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: page
+title: AgoraDesk
+subtitle: Peer-to-peer cryptocurrency marketplace
+#cover-img: /assets/img/path.jpg
+share-img: https://bitcoiner.guide/assets/img/QNA_SHARE_IMG.png
+---
+<p align="center">
+  <a href="#what-is-agora-desk">Intro</a> |
+  <a href="#before-you-start">Preparation</a> |
+  <a href="#buying-bitcoin">Buy Sats</a> |
+  <a href="#faq">FAQ</a> |
+  <a href="#learn-more">Learn More</a> 
+</p>
+
+<p align="center">
+<img src="/assets/img/agoradesk.png" class=responsive style="max-width: 80%; height: auto;" />
+</p>
+
+## Introduction
```

Comment:
Can we simplify this or change it slightly just so it's not the same as shown in the other noKYC guides?

Reply:
Sure thing, I will give it a try and make this part different from other guides.